contracts/examples/Template.sol

```diff
@@ -120,6 +125,10 @@ contract Template is TemplateBase {
         acl.createPermission(root, lock, lock.CHANGE_DURATION_ROLE(), voting);
         acl.createPermission(root, lock, lock.CHANGE_AMOUNT_ROLE(), voting);
 
+        acl.createPermission(this,lock,lock.LOCK_TOKENS_ROLE(), this);
+        // creating param for dissent oracle (the idea here is when an entity tries to mint tokens, it first checks with the oracle)
```

Comment:
Won't this Oracle be a Token Manager Membership Oracle? As opposed to the Dissent Oracle.

Reply:
Yup, copied it from the one i did in the dissent and totally forgot to change it.